pjsip/include/pjsua-lib/pjsua.h

```diff
@@ -4782,6 +4782,21 @@ typedef struct pjsua_acc_config
      */
     pj_bool_t        use_shared_auth;
 
+    /**
+    * Specify how to automatically process incoming SIP MESSAGE requests.
+    *
+    * When set to PJ_TRUE (automatic processing), incoming MESSAGE requests
+    * will be responded to immediately with SIP 200 OK (legacy behavior).
+    *
+    * When set to PJ_FALSE (manual processing), incoming MESSAGE requests
+    * will create UAS transactions and defer response sending. Applications
+    * must use pjsua_acc_send_response() to send responses manually.
+    *
+    * Default: PJ_TRUE (automatic response for backward compatibility)
+    */
+    pj_bool_t        auto_process_sip_message;
```

Comment:
IMHO `auto_process` sounds somewhat ambigous as the libray is merely sending a response automatically, not fully processing it, so how about `auto_respond` instead?